frontend/src/containers/Issue/index.js

```diff
@@ -0,0 +1,68 @@
+import React, {Component} from 'react'; // eslint-disable-line no-unused-vars
+import IssueForm from '../../components/IssueForm'; // eslint-disable-line no-unused-vars
+import Uploader from '../../components/Uploader';
+import {getIssue} from './actions';
+import './style.css';
+
+export default class Issue extends Component {
+  state = {
+    issue: {_id:0},
+    uploader_visible: false
+  };
+  componentWillMount = () => {
+    getIssue()
+    .then( (response) => {
+      this.setState( { issue: response});
+    })
+    .catch( (err) => {
+      console.error( "getIssue failed:", err); // eslint-disable-line no-console
+    });
+  };
+  issueFormSubmit = (e) => {
+    console.log( "issue form submit issue:", this.state.issue);
+  };
+  onFieldChange = (e) => {
+    console.log( "field changed:", e.target.name, e.target.value);
+    const {issue} = this.state;
+    issue[e.target.name] = e.target.value;
+    this.setState( { issue});
+  };
+  toggleUploaderViz = () => {
+    this.setState( { uploader_visible: !this.state.uploader_visible});
+  };
+
+  uploadImage = ( files) => {
```

Comment:
why the spacing ?

Reply:
... and the unecessary brackets. I've edited them all out